app/models.py

```
@@ -1,6 +1,7 @@
 """Data models for the Speaking Meeting Bot API."""
 
 from typing import Any, Dict, List, Optional
+from datetime import datetime
```

Ruff (0.8.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)